### PR TITLE
✨ add an image-based renderer

### PR DESCRIPTION
**Pull request by garak**
This can be an interesting feature

### CODE REVIEW

**Comment by garak** [comment]
Maybe this can improved by adding coordinates on borders... WDYT @akondas?

**Review by akondas — APPROVED** [review]
:exploding_head: :heart_eyes:  WOW, this already looks really good. Thanks for that :+1: :beers: 

image

**Comment by akondas** [comment]
> Maybe this can improved by adding coordinates on borders... WDYT @akondas?

Looks like a good idea. Maybe we could also add some default/embedded images that could be used?

File: buddy.yml [hunk 1]
```diff
@@ -17,9 +17,11 @@
             - "bash <(curl -s https://codecov.io/bash)"
         setup_commands:
             - "echo \"memory_limit=-1\" >> /usr/local/etc/php/conf.d/buddy.ini"
-            - "apt-get update && apt-get install -y git zip"
+            - "apt-get update && apt-get install -y git zip libfreetype6-dev libjpeg62-turbo-dev libpng-dev"
             - "curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer"
             - "pecl install pcov && docker-php-ext-enable pcov"
+            - "docker-php-ext-configure gd --with-freetype --with-jpeg"
+            - "docker-php-ext-install gd"
```
Comment (akondas) [inline buddy.yml:24]: :+1: thanks for that

**Comment by garak** [comment]
> Looks like a good idea. Maybe we could also add some default/embedded images that could be used?

I'm a bit sceptical about including images in this library... after all this renderer is optional, and we risk to add some burden.
Maybe we can create a separate repository for resources?

**Comment by akondas** [comment]
They don't even have to be pictures, just a script that downloads them? Or possibly a link to pictures / script in the documentation?

**Pull request merged**